README.md

```diff
@@ -1,27 +1,42 @@
-# WebAssembly Community Group: Design
+# WebAssembly Design
 
-This repository contains documents describing the high-level design of
-WebAssembly and is part of the 
-[WebAssembly Community Group](https://www.w3.org/community/webassembly/).
+> This repository contains documents describing the design and high-level overview of WebAssembly.
 
-The actual WebAssembly specification is being developed in the
-[spec repository](https://github.com/WebAssembly/spec/). High-level design
-discussions should continue to be held in the design repository for now so
-that the spec work can remain focused.
+> The documents and discussions in this repository are part of the [WebAssembly Community Group](https://www.w3.org/community/webassembly/).
 
-**Expect the contents to be in flux: everything is still tentative.**
+## Overview
 
-A good starting point is the [high-level design goals](HighLevelGoals.md),
-followed by the [FAQ](FAQ.md).
+WebAssembly or wasm is a portable, low-level, binary-format abstract syntax tree designed as an efficient target language for compilers.
```

Comment:
@titzer Well that is a huge problem. If you all consider the AST 'an implementation detail' then can the web community also take it that you consider wasm being 'source code' is a non-core requirement? Personally I don't want the web having a binary machine code (especially when it can have a primitive source code), and having an AST (which is basically saying it has expressions) is basically saying it could be a linear machine code. You all went off and colluded on this without consulting the wider web community while some of us were working hard to move general JS performance forward and while you we stalling advances, and I even object to the naming of this as 'Assembler'. You have basically got nowhere on the performance front over the last year (except for the prospect of reducing the JS parse time by having the AST) - the AST is all this project really is, and you don't even want to mention it.

Accepting support coming from the same small group, the same group that colluded on this in private to start with disregarding the views of the web community, should not stand in an open web community forum to override objections and alternative views. If you want to take it for a vote to the community group, and if they vote for a web machine code rather than a primitive source code then I give up, but not until then.